includes/polling/os/eatonups.inc.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Please convert this to the new os format, https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery

Reply:
@Jellyfrog I do not understand.

Discovery *is* performed with a yaml file. Polling is defined with php as everything else. Please elaborate.

Reply:
these values now belong to the discovery phase, please check https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery for more info.  
Either the php way; https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Qnap.php  
or the yaml way;  https://github.com/librenms/librenms/blob/master/includes/definitions/discovery/teldat.yaml

Reply:
Thank you. Will try again.